tests/bud.bats

```diff
@@ -2422,15 +2431,15 @@ _EOF
 }
 
 @test "bud COPY with URL should fail" {
-  mkdir ${TESTSDIR}/bud/copy
-  FILE=${TESTSDIR}/bud/copy/Dockerfile.url
+  mkdir ${TESTDIR}/budcopy
+  FILE=${TESTDIR}/budcopy/Dockerfile.url
   /bin/cat <<EOM >$FILE
 FROM alpine:latest
 COPY https://getfedora.org/index.html .
 EOM
 
-  run_buildah 125 build --signature-policy ${TESTSDIR}/policy.json -t ${target} -f ${TESTSDIR}/bud/copy/Dockerfile.url ${TESTSDIR}/bud/copy
-  rm -r ${TESTSDIR}/bud/copy
+  run_buildah 125 build --signature-policy ${TESTSDIR}/policy.json -t foo -f ${TESTDIR}/budcopy/Dockerfile.url
+  expect_output --substring "error building .* source can.t be a URL for COPY"
```

Comment:
@containers/buildah-maintainers PTAL at this. This test was completely broken, and I want to use it as an emphatic indicator of why we need to check error messages.

The bug was in `-t ${target}`. `$target` was never defined, so this became `-t -f ...Dockerfile etc` which led to this failure:
```
tag -f: invalid reference format
```
So, yeah, the exit-status check was correct but for completely wrong reason.

This is not a shaming attempt. I promise I have not run `tig blame`, and beg all of you not to do so either; and if this is your code, don't speak up. This is a bug that can happen to any of us, and has. The point is, pleeeeeease don't just rely on exit-status checks. Please look for a distinct error message. Thank you.